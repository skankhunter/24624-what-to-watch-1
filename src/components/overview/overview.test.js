import React from "react";
import renderer from "react-test-renderer";

import Overview from "./overview.jsx";

const mocks = {
  activeFilm: {
    backgroundImage: `back`,
    description: `description`,
    director: `Director`,
    genre: `Action`,
    id: 1,
    isFavorite: false,
    name: `Title`,
    poster: `string`,
    posterImage: `string`,
    preview: `string`,
    rating: 5,
    released: 2018,
    runTime: 88,
    scoresCount: 2000,
    starring: [`1`, `2`, `3`],
    videoLink: `link`
  }
};

describe(`Overview:`, () => {
  it(`Correctly renders after relaunch`, () => {
    const tree = renderer.create(<Overview {...mocks} />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
