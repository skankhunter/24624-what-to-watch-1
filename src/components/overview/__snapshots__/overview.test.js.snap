// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overview: Correctly renders after relaunch 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      5
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        normal
      </span>
      <span
        className="movie-rating__count"
      >
        2000
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      description
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        1, 2, 3
      </strong>
    </p>
  </div>,
]
`;
